### PR TITLE
Disable AWX support

### PR DESCRIPTION
**Pull request by dwrightco1**
Removed AWX flags from pf9-express usage and argument parsing code, but left functions that manage AWX in code base.

### CODE REVIEW

File: lib/pf9-builder/testcase-01-centos.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## openstack
```
Comment (jb4free) [inline lib/pf9-builder/testcase-01-centos.csv]: Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore

File: lib/pf9-builder/testcase-01-ubuntu.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## openstack
```
Comment (jb4free) [inline lib/pf9-builder/testcase-01-ubuntu.csv]: Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore

File: lib/pf9-builder/testcase-02-centos.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## kubernetes
```
Comment (jb4free) [inline lib/pf9-builder/testcase-02-centos.csv]: Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore

File: lib/pf9-builder/testcase-02-ubuntu.csv [hunk 4]
```diff
@@ -0,0 +1,6 @@
+## kubernetes
```
Comment (jb4free) [inline lib/pf9-builder/testcase-02-ubuntu.csv]: Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore

File: lib/pf9-builder/testcase-03-ubuntu.csv [hunk 5]
```diff
@@ -0,0 +1,10 @@
+## kubernetes
```
Comment (jb4free) [inline lib/pf9-builder/testcase-03-ubuntu.csv]: Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore

**Comment by mmccarre** [comment]
Closing this request, moving to pull request #144 for cleaner merge/review

**Pull request closed without merging**